edkrepo_manifest_parser/edk_manifest.py

```diff
@@ -648,6 +688,43 @@ def __eq__(self, other):
     def __ne__(self, other):
         return not self.__eq__(other)
 
+class _PatchSet():
+    def __init__(self, element):
+        try:
+            self.remote = element.attrib['remote']
+            self.name = element.attrib['name']
+            self.parentSha = element.attrib['parentSha']
+            self.fetchBranch = element.attrib['fetchBranch']
+        except KeyError as k:
+            raise KeyError(REQUIRED_ATTRIB_ERROR_MSG.format(k, element.tag))
+
+    @property
+    def tuple(self):
+        return PatchSet(self.remote, self.name, self.parentSha, self.fetchBranch)
+
+class _PatchSetOperations():
+    def __init__(self, element):
+        self.type = element.tag
+        try:
+            self.file = element.attrib['file']
+        except KeyError as k:
+            self.file = ''
+        try:
+            self.sha = element.attrib['sha']
+        except KeyError as k:
+            self.sha = ''
+        try:
+            self.sourceRemote = element.attrib['sourceRemote']
+        except KeyError as k:
+            self.sourceRemote = ''
+        try:
+            self.sourceBranch = element.attrib['sourceBranch']
+        except KeyError as k:
+            self.sourceBranch = ''
+

```

Comment:
Instead of setting these values to an empty string lets set them to None

To keep alignment with the style precedent used else where lets use source_remote instead of sourceRemote etc... for the members of this class. Note that this does not have to match the syntax we are searching for in the manifest file.